src/format/opamFile.ml

```diff
@@ -3203,6 +3203,60 @@ module OPAMSyntax = struct
     in
     Pp.pp parse print
 
+  let rewrite_xfield = "x-allow-env-path-rewrite-on-windows"
+  let handle_env_paths =
+    let pp_constraint =
+      pp_minimal_opam_version (OpamVersion.of_string "2.2")
```

Comment:
Not sure if we need that constraint, as it won't be syntaxically broken if read by opam < 2.2, just may not work on windows, what is the current status of 2.1.

Reply:
From dev meeting: remove this constraint, have a case by case availability guard in repo opam files.